### PR TITLE
ci(deploy): Add Ramsalt related installer types and paths

### PR DESCRIPTION
**Pull request by doxigo**
To support Ramsalt themes and modules

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -66,6 +66,17 @@
         ]
     },
     "extra": {
+        "installer-types": ["bower-asset", "npm-asset", "ramsaltmedia-module", "ramsaltmedia-theme"],
```
Comment (szeidler) [inline composer.json:73]: It's doubling the installer-paths. Could you check this?

**Review by szeidler — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (doxigo) [reply composer.json:73]: So you're suggesting to mix this into `installer-paths` directly? like `"web/themes/ramsaltmedia/{$name}": ["type:ramsaltmedia-module"],` ?

Comment (szeidler) [reply composer.json:73]: See the resulting composer.json. It has two installer paths sections then, which is wrong.
Bildschirmfoto 2022-02-11 um 15 19 13

Comment (doxigo) [reply composer.json:73]: Idk how that even happened but removed the obviously older `installer-paths` at the bottom

**Review by szeidler — APPROVED** [review]

**Pull request merged**